manifests/v/vatis-project/vAtis/4.1.0-beta.1/vatis-project.vAtis.locale.en-US.yaml

```
@@ -0,0 +1,17 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: vatis-project.vAtis
+PackageVersion: 4.1.0-beta.1
+PackageLocale: en-US
+Publisher: vATIS for VATSIM
+PublisherUrl: https://github.com/vatis-project
```

```suggestion
PublisherUrl: https://github.com/vatis-project
PublisherSupportUrl: https://discord.gg/rTMZhjhu9Q
```

Extracted from [Getting Help | vATIS for VATSIM](https://vatis.app/docs/getting-help)